Sources/BraintreeCore/BTAPIClient.m

```diff
@@ -90,7 +90,8 @@ - (nullable instancetype)initWithAuthorization:(NSString *)authorization sendAna
             configurationCache = [[NSURLCache alloc] initWithMemoryCapacity:1 * 1024 * 1024 diskCapacity:0 diskPath:nil];
         });
         configuration.URLCache = configurationCache;
-        configuration.requestCachePolicy = NSURLRequestReturnCacheDataElseLoad;
+        // Use the caching logic defined in the protocol implementation, if any, for a particular URL load request.
+        configuration.requestCachePolicy = NSURLRequestUseProtocolCachePolicy;
```

Comment:
random q: does config endpoint enforce caching with HTTP headers?

Reply:
Do you mean like passing a `Cache-Control` header? If so the preferred method for `NSURLSession` is to use the `requestCachePolicy` on the configuration as there is some weirdness with headers not being respected (for example `NSURLSession` also ignores `Keep-Alive` headers).

Reply:
Right yeah I guess that's a question for the gateway actually. If config has caching behavior built in we could leverage that on Android too.

Reply:
Yeah certainly something to look into for both SDKs